Automattic-Tracks-iOS/TracksService.m

```diff
@@ -54,7 +54,7 @@ @interface TracksService ()
 NSString *const TracksUsernameKey = @"_ul";
 
 NSString *const TracksUserTypeAnonymous = @"anon";
-NSString *const TracksUserTypeWPCOM = @"wpcom:user_id";
+NSString *const TracksUserTypeAuthenticated = @"wpcom:user_id";
```

Comment:
Is this new const string going to be used when tracking events for simplenote authenticated user? If so, there is a problem with it. We still need to decide which string will represent the Simplenote user on the server side.